test/path.jl

```diff
@@ -68,6 +68,15 @@ for S in (String, GenericString)
     @test joinpath(splitdir(S(homedir()))...) == homedir()
     @test string(splitdrive(S(homedir()))...) == homedir()
 
+    if is_windows()
+        @test splitdrive("\\\\servername\\hello.world\\filename.ext") ==
```

Comment:
this is inside a loop over `S` but you aren't using `S` like all the other tests here do (testing either String or GenericString as an approximation of unicode handling)